### DISCUSSION TITLE
Storybook HowTo

### DISCUSSION THREAD

**Discussion by pfried** [vercel/turborepo#289] [Ideas]
I am just getting started with turborepo, so naturally I have some questions / thoughts

Currently we have a lot of packages in their own repos but few people working on them, so to me it seems to be a better solution to create a monorepo to tame the complexity of the packages.

We have a storybook in both our marketing website and our app. It would be great to get this stuff out of those packages since they clutter these apps quite a bit and would allow us to centralize docs in general.

Is there any recommendation or example on how to set this up? Can Anyone provide an example of a `next.js` app with `storybook` (and ideally tailwind 😄 )

Related https://github.com/vercel/turborepo/issues/209#issue-1077875521

**Comment by anthonyshew** [comment]
Check out [the Design System example](https://github.com/vercel/turbo/tree/main/examples/design-system) for Storybook and [the Tailwind example](https://github.com/vercel/turbo/tree/main/examples/with-tailwind) for Tailwind!
